entity_event/tests/context_loader_tests.py

```diff
@@ -271,12 +271,12 @@ def test_dict_find_deep_nested_list_key(self):
         self.assertEquals(list(context_loader.dict_find(larger_dict, 'key')), [(d, ['value']), (d, ['value'])])
 
 
-class GetModelIdsToFetchTest(SimpleTestCase):
+class GetModelIdsToFetchTest(TestCase):
     def test_no_events(self):
         self.assertEquals(context_loader.get_model_ids_to_fetch([], {}), {})
 
     def test_no_context_hints(self):
-        e = N(models.Event, context={}, persist_dependencies=False)
+        e = G(models.Event, context={})
```

Comment:
you'll just need to add id=1 to those N models. Can you do that so that the tests dont hit the db?